src/components/Account/OAuthLogin/OAuthLogin.container.js

```diff
@@ -54,11 +59,20 @@ class OAuthLoginContainer extends React.Component {
     return !!user.email;
   }
 
+  handleError() {
+    const { history } = this.props;
+    this.setState({ hasErrors: true });
+    setTimeout(() => {
+      history.replace('/');
```

Comment:
Probably would be better to redirect the user to /login-signup if an error occurred